include/core/jtrace.h

```diff
@@ -58,13 +58,18 @@ void j_trace_leave (JTrace*);
 
 G_DEFINE_AUTOPTR_CLEANUP_FUNC(JTrace, j_trace_leave)
 
+#ifndef JULEA_DEBUG
```

Comment:
This seems to be the wrong way around. We do want tracing for debug builds, right?

Reply:
yes, that should be #ifdef